Ghidra/Features/Base/src/main/java/ghidra/app/util/opinion/OmfLoader.java

```diff
@@ -461,7 +533,7 @@ private boolean createSymbol(OmfSymbol symbol, Address address, SymbolTable symb
 			}
 			catch (InvalidInputException e) {
 				log.appendMsg("Unable to create symbol " + symbol.getName() + " at 0x" +
-					Long.toHexString(address.getOffset()));
+						Long.toHexString(address.getOffset()));
```

Comment:
Something might be off with your formatter.  Another indent should not be added here.  Are you using our eclipse formatter?

Reply:
I have used the standard formatter, so not sure what has gone wrong.